MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapter.cs

```diff
@@ -17,12 +17,22 @@ public interface IMvxRecyclerAdapter
         [MvxSetToNullAfterBinding]
         IEnumerable ItemsSource { get; set; }
 
-        IMvxTemplateSelector ItemTemplateSelector { get; set; }
+        MvxBaseTemplateSelector ItemTemplateSelector { get; set; }
```

Comment:
@kjeremy sorry, we can't keep this interface. It's impossible to implement current header/footer/grouping template selector support with keeping that as interface. To be honest it should not even exist as it was "redudant" to MvxTemplateSelector. 
That's the only change that might break... however I don't even think anyone has even used that - everyone used MvxTemplateSelector class. The only place where it might have been used is custom IMvxRecyclerAdapter.